sympy/functions/elementary/exponential.py

```diff
@@ -645,6 +645,37 @@ def fdiff(self, argindex=1):
         else:
             raise ArgumentIndexError(self, argindex)
 
+    def _eval_derivative(self, s):
```

Comment:
Redefining this makes `fdiff` above effectively dead code.

Reply:
I'll revert this and patch heurisch instead.